packages/command/src/sfdxFlags.ts

```diff
@@ -598,8 +599,9 @@ function validateCustomFlag<T>(key: string, flag: flags.Any<T>): flags.Any<T> {
 export function buildSfdxFlags(
   flagsConfig: FlagsConfig,
   options: { targetdevhubusername?: boolean; targetusername?: boolean }
+  // tslint:disable-next-line:no-any matches oclif
 ): flags.Output {
-  const output: flags.Output = {};
+  const output: Dictionary<flags.Any<unknown>> = {};
```

Comment:
This turned out to be unnecessary but still adds a bit of clarity to the fn impl.  I had hoped to return this type in the method signature, but it wasn't compatible with the base class.